### ISSUE TITLE
tf.nn.batch_normalization and tf.keras.layers.BatchNormalization outputs gives a diff (10^-2) range

### ISSUE THREAD

**Issue by yogeesh-alphaics** [tensorflow/tensorflow#52882]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): 20.04

I have implemented a classification model , which has Batchnormalization layers into it. While training the model , i used ```tf.keras.layers.BatchNormalization``` layer for training. Then manually extracted the bn weights (gamma, beta, mean , var) from checkpoint file and called the bn using ```tf.nn.batch_normalization``` layer using above statistics for inference. I observed a drop in testing accuracy (~75%) as compared to the scenario where i load the weights in the model using keras itself ```model.load_weights(chkpnt_file)``` (~92%).   
Though if i replace the ```tf.nn.batch_normalization``` call to keras bn layer call by calling ```set_weights``` function , both the cases gived 0.0 diff, which apprently led me to beleive that there is diff in both the calls while the input and bn_weights are exactly same. epsilon is also same.  

**Standalone code to reproduce the issue**
Please refer to this colab notebook(https://colab.research.google.com/drive/1nysYyCR5Ay_jkPlh1WD9A1Mf01eVvhMu? usp=sharing) for a small example. Here the error is coming to be a margin of 10^-2 (which i think is also big given all inputs are same) , but in my project there is huge diff.

**Comment by sushreebarsa** [comment]
@yogeesh-alphaics 
Please post this issue on [keras-team/keras repo.](https://github.com/keras-team/keras/issues)
To know more see;
[https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999)
Thank you!

**Comment by yogeesh-alphaics** [comment]
okay thanks for the update @sushreebarsa
